### DISCUSSION TITLE
Not normal floating (glm)

### DISCUSSION THREAD

**Discussion by wkitlasten** [usgs/pestpp#244] [General]
I'm doing a quick glm run to try to get a "better" starting point before my ies run. 263 pars are listed under the following errors. What are the implications of this? Should I be worried?

Maybe something wrong with my jcb? 
Parval1, parbnds, and derinc all seem legit and jcb.isnull().any().any()=False, but maybe I need to adjust those a little?
Maybe I need to set lambda bigger (smaller)? Maybe something more insidious in my builds?

Any other suggestions or insights appreciated.

```
  computing upgrade vector (lambda = 0.1)  1 / 5
not normal floating point in upgrade pars: PNAME:INFLOW_INST:0_PTYPE:GR_USECOL:2_PSTYLE:M_IDX0:4066
...
  computing upgrade vector (lambda = 1)  2 / 5
not normal floating point in upgrade pars: PNAME:INFLOW_INST:0_PTYPE:GR_USECOL:2_PSTYLE:M_IDX0:4066
...
```

**Comment by jtwhite79** [comment]
Hmm.  Usually this happens with log pars when the upgrade is extreme (large residuals) and lots of singular components are being used.  You can try smaller maxsing and or larger eigthresh- that will usually stabilize things.  Can you send me your master dir so I can have a look?

**Comment by wkitlasten** [comment]
Doh, I forgot to unweight my extreme obs ('less_') that I was using to drop bad reals in ies. Error makes sense now. Sorry.
